ginko/src/dts/parser.rs

```diff
@@ -9,9 +9,14 @@ use crate::dts::reader::{ByteReader, Reader};
 use crate::dts::tokens::{CompilerDirective, Lexer, PeekingLexer, Reference, Token, TokenKind};
 use crate::dts::HasSpan;
 use itertools::Itertools;
-use std::path::Path as StdPath;
+use std::path::{Path as StdPath, PathBuf};
 use std::sync::Arc;
 
+#[derive(Clone, Default)]
```

Comment:
You can ignore this comment if you want but maybe it's nice to add some default include paths, i.e., `/usr/include`, `/usr/lib/include` and similar for Linux (there are probably some sensible default options for Windows and macOs as well).

Reply:
I'm not sure if adding default includes might introduce some unwanted implicity. It also raises the question what happens if `--include` is provided, are the defaults ignored or merged? I also couldn't find defaults in `dtc`

Reply:
Alright, I reckon the way to go is what you initially implemented then. We can always change the behaviour later, should the desire arise.